pkg/operator/controller.go

```diff
@@ -107,7 +110,12 @@ func AddToManager(m, rootM manager.Manager, explicitKubeconfig string, coreClien
 			if infraStatus.PlatformStatus == nil || infraStatus.PlatformStatus.GCP == nil {
 				log.Fatalf("missing GCP configuration in platform status")
 			}
-			a, err = gcpactuator.NewActuator(m.GetClient(), rootM.GetClient(), infraStatus.PlatformStatus.GCP.ProjectID)
+			enabledFeatures, err := util.GetEnabledFeatureGates([]configv1.FeatureGateName{features.FeatureGateGCPCustomAPIEndpoints})
+			if err != nil {
+				return fmt.Errorf("error getting enabled feature gates: %v", err)
+			}
+			gcpCustomEndpointsEnabled, _ := enabledFeatures[features.FeatureGateGCPCustomAPIEndpoints]
```

Comment:
In conjunction with the comment in pkg/util/featuregate.go, modify this to get currentFeatureGates, and then check for gcpCustomEndpointsEnabled separately.
```
	currentFeatureGates, err := util.GetCurrentFeatureGates()
	if err != nil {
		return err
	}
```
	
That block should be included in the prior commit so it remains after the featuregate is removed. And then for this specific feature gate, you would add
```
	gcpCustomEndpointsEnabled := currentFeatureGates.Enabled(features.FeatureGateGCPCustomAPIEndpoints)
```